settings.gradle.kts

```diff
@@ -5,8 +5,7 @@ plugins {
 
 rootProject.name = "native-platform"
 
-include("testApp")
-
-project(":testApp").projectDir = File(rootDir, "test-app")
+include("test-app")
+include("native-platform")
```

Comment:
Does this mean that we now have two projects called `native-platform`? Is that a problem?

Reply:
The root project name seems to be a different beast as the other project names. The path for the root project is still `:`, so I don't think this is a problem.

@jjohannes do you know if this could cause problems?

Reply:
I am not aware of any problems.

We only sometimes have issues if two projects have the same "simple name" if they do stuff based on that name only (like naming a jar file, or putting it into the GAV coordinate).

But since the root project intentionally does "nothing", I don't think it is a problem.